### PR TITLE
Update to R# 2018.3

### PR DESCRIPTION
**Pull request by ac931274**
Please find attached updates to allow ForTea to work with R# 2018.3.  These work on my system (VS 2016 Enterprise)  I am testing on others but it appears to be working.

### CODE REVIEW

File: build/.build.csproj [hunk 1]
```diff
@@ -12,9 +12,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nuke.Common" Version="$(NukeVersion)" />
+    <PackageReference Include="NuGet.CommandLine" Version="4.9.2">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Nuke.Common" Version="0.4.0" />
```
Comment (ac931274) [inline build/.build.csproj]: In hindsight the previous version of this line is probably the better one to use.

File: build/.build.csproj [hunk 2]
```diff
@@ -12,9 +12,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nuke.Common" Version="$(NukeVersion)" />
```
Comment (ac931274) [inline build/.build.csproj]: Don't remove this line, since it uses a param.

**Comment by LeEmo86** [comment]
It would be great to get this in, I'd love to be able to use this extension again!

**Comment by llfgmb** [comment]
Can I have some instructions how to compile and install the extension from my locally disk?

**Comment by ac931274** [comment]
@llfgmb Hi there, should @MrJul not reply by the end of the month, based on his previous comments about being very busy, I will publish my version on JetBrains plugin site, with the understanding that it will be taken down as soon as 1) @MrJul takes my pull request or 2) he requests to me that it be taken down.

**Comment by llfgmb** [comment]
Thank you. Is it possible to describe what to do with built libriary? I can't find information how to install it in my R#.

**Comment by remcoros** [comment]
ReSharper options -> Env. -> Extension Manager

Add a new source to a local folder. Place the "ForTea....nupkg" there.

It should now show up in ReSharper extension manager.

**Comment by MrJul** [comment]
Hello and thank you for your work!
I'm merging, checking then publishing the new version.

**Pull request merged**